Formula/bettercap.rb

```diff
@@ -0,0 +1,31 @@
+class Bettercap < Formula
+  desc "Complete, modular, portable and easily extensible MITM framework"
+  homepage "https://www.bettercap.org/"
+  url "https://github.com/evilsocket/bettercap/archive/v1.6.2.tar.gz"
+  sha256 "1b364d7e31be5fa7b5f93eefe76763ad7bd4ac0b7b6bb4af05483157580a9cb9"
+
+  depends_on "openssl"
+  depends_on :ruby => "2.2.2"
```

Comment:
I don't see this specified as a minimum version on the homepage; it just says 1.9 or greater.

Reply:
I think the minimum version is outdated.

```
ERROR:  Error installing bettercap:
	packetfu requires Ruby version >= 2.1.0.
```
```
ERROR:  Error installing bettercap:
	nio4r requires Ruby version >= 2.2.2.
```

Reply:
Gotcha.  If you have a moment it would be nice to have that reported upstream.